script/hassfest/manifest.py

```diff
@@ -29,6 +29,8 @@
         vol.Required("dependencies"): [str],
         vol.Optional("after_dependencies"): [str],
         vol.Required("codeowners"): [str],
+        vol.Optional("logo"): str,
```

Comment:
Should we specify it it's a url or a file path ?

Reply:
Yes, see comment above :)